Aptoide/aptoide-client-v8
# [ASV-1429] Increased mopub sdk version

**What does this PR do?**

   This PR aims at increasing the mopub sdk version. With this new version, there are some changes regarding the way to initialize the non-partner networks sdk's.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

Check if you can correctly see ads on all of the production placements (home, search and appview) for all ad types (native, banners and interstitial).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1429](https://aptoide.atlassian.net/browse/ASV-1429)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
